### PR TITLE
Pd 3567 sidebar dropdown

### PR DESCRIPTION
**Pull request by aljordan82**
## What's this PR do?
Made the Filters a dropdown in NOU, PRM, and  PA.
## How should this be manually tested?
The sidebar filters should become a dropdown in mobile view.
### Front-End

- [ ] Works on IE 11 and Edge
- [ ] Works on Mobile
- [ ] Unit tests added or ticket exists to add tests

### CODE REVIEW

**Comment by koonsies** [comment]
This seems to be the case in both mobile and regular view, but can we fix the spacing on the filters? Looks.. smushed on NUO and PA.

screen shot 2017-09-26 at 2 34 39 pm
screen shot 2017-09-26 at 2 31 55 pm

File: gulp/src/nonuseroutreach/reducers/navigation-reducer.js [hunk 1]
```diff
@@ -1,17 +1,5 @@
...
-      'have your employees send outreach emails. ',
-    'User sending email to these addresses will receive a ' +
-      'confirmation message upon successfully submitting a Non-User Outreach record.',
-  ],
-  records: ['Use this page to view outreach records from non-users.'],
-};
-
 export const initialNavigation = {
   currentPage: 'overview',
   tips: [],
```
Comment (koonsies) [inline gulp/src/nonuseroutreach/reducers/navigation-reducer.js]: tips still part of initial data

**Comment by koonsies** [comment]
Tests failing. Looks like a test checks for tips.

**Comment by aljordan82** [comment]
@koonsies Labels css is fixed, wrong classes were used.

**Pull request merged**